packages/demo-app/src/examples/TooltipExample.tsx

```diff
@@ -31,7 +31,6 @@ export const TooltipExample = React.memo(() => {
                         Always open tooltip
                     </span>
                 }
-                isOpen={true}
```

Comment:
Discovered this while running demo-app. This appears to be crashing things. possibly related to #7179?

![CleanShot 2025-01-17 at 16 23 16@2x](https://github.com/user-attachments/assets/2af1c7ea-6294-43f4-b40e-ef6c14099015)